lib/vehicle.rb

```diff
@@ -22,4 +24,16 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def plate_type
+    if engine == :ev
+      :ev
+    elsif engine == :ice && antique?
```

Comment:
```suggestion
    elsif antique?
```
I would probably say it's okay to just check `antique?` here since in our imaginary app there's only two engine types, however I can definitely follow the line of thought that it's not necessarily the case.